PyTorch/Getting_Started_with_vLLM/Getting_Started_with_vLLM.ipynb

```diff
@@ -0,0 +1,848 @@
+    "\n",
+    "It is highly recommended to use the latest Docker image from Intel Gaudi vault. Refer to the [Run Docker Image](https://github.com/HabanaAI/vllm-fork/blob/habana_main/README_GAUDI.md#run-docker-image) section from [Intel Gaudi documentation](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#pull-prebuilt-containers) for more details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b185ee14-ec2f-42ac-a756-7046e9661efd",
+   "metadata": {},
+   "source": [
+    "The following cells install vLLM for Gaudi required to run the contents of this notebook. For more information on installing vLLM for Gaudi refer to [Build And Install vLLM](https://github.com/HabanaAI/vllm-fork/blob/habana_main/README_GAUDI.md#build-and-install-vllm-fork)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dfaf25e3-3961-4271-8682-025f863cd7c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "git clone https://github.com/HabanaAI/vllm-fork.git\n",
+    "cd vllm-fork\n",
+    "git checkout v0.4.2-Gaudi-1.16.0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "38905bdd-9982-4653-b8ed-ef100ba4d68a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!docker run -d \\\n",
+    "  --runtime=habana \\\n",
+    "  -v $(pwd):/app \\\n",
+    "  -e HABANA_VISIBLE_DEVICES=0 \\\n",
+    "  -e OMPI_MCA_btl_vader_single_copy_mechanism=none \\\n",
+    "  --cap-add=sys_nice \\\n",
+    "  --net=host \\\n",
+    "  --ipc=host \\\n",
+    "  --name=vllm_installation \\\n",
+    "  vault.habana.ai/gaudi-docker/1.16.2/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest \\\n",
+    "  /bin/bash -c \"cd /app/vllm-fork && pip install -e . && pip install flask && echo $'\\n\\nInstallation completed successfully'\"  # This may take 5-10 minutes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e7e824b-9eda-4f00-9c8e-1901a75d6526",
+   "metadata": {},
+   "source": [
+    "## Getting Started with vLLM"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1705acf-def1-4a11-87e5-94bf7a5c8e8e",
+   "metadata": {},
+   "source": [
+    "In this tutorial section below topics will be covered on how to setup and run on Intel Gaudi\n",
+    "- Check vLLM installation\n",
+    "- Prerequistes \n",
+    "- Run vLLM for Offline batched Inference\n",
+    "- Deploy vLLM via Flask \n",
+    "- Deploy vLLM for Online Inference via OpenAI-Compatible Server"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebcd038a-46b2-423a-9342-64a1ce248831",
+   "metadata": {},
+   "source": [
+    "### Check vLLM installation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f765dcbf-5e6b-4102-9417-f799872acdcb",
+   "metadata": {},
+   "source": [
+    "Check logs from the docker container to see if vLLM and flask have been installed successfully."
```

Comment:
Docker logs command does not show this message. Instead the below is seen:
Obtaining file:///app/vllm-fork
  Installing build dependencies: started

Reply:
the appropriate message appears after a few minutes. Need to add a Note asking user to wait for a couple of minutes before running this cell.